packages/codemirror/typings/codemirror/codemirror.d.ts

```diff
@@ -100,891 +52,10 @@ declare namespace CodeMirror {
-    addLineWidget(
-      line: any,
-      node: HTMLElement,
-      options?: {
-        /** Whether the widget should cover the gutter. */
-        coverGutter: boolean;
-        /** Whether the widget should stay fixed in the face of horizontal scrolling. */
-        noHScroll: boolean;
-        /** Causes the widget to be placed above instead of below the text of the line. */
-        above: boolean;
-        /** When true, will cause the widget to be rendered even if the line it is associated with is hidden. */
-        showIfHidden: boolean;
-      }
-    ): CodeMirror.LineWidget;
-
-    /** Programmatically set the size of the editor (overriding the applicable CSS rules).
-        width and height height can be either numbers(interpreted as pixels) or CSS units ("100%", for example).
-        You can pass null for either of them to indicate that that dimension should not be changed. */
-    setSize(width: any, height: any): void;
-
-    /** Scroll the editor to a given(pixel) position.Both arguments may be left as null or undefined to have no effect. */
-    scrollTo(x: number, y: number): void;
-
-    /** Get an { left , top , width , height , clientWidth , clientHeight } object that represents the current scroll position, the size of the scrollable area,
-     and the size of the visible area(minus scrollbars). */
-    getScrollInfo(): {
-      left: any;
-      top: any;
-      width: any;
-      height: any;
-      clientWidth: any;
-      clientHeight: any;
-    };
-
-    /** Scrolls the given element into view. pos is a { line , ch } position, referring to a given character, null, to refer to the cursor.
-     The margin parameter is optional. When given, it indicates the amount of pixels around the given area that should be made visible as well. */
-    scrollIntoView(pos: CodeMirror.Position, margin?: number): void;
-
-    /** Scrolls the given element into view. pos is a { left , top , right , bottom } object, in editor-local coordinates.
-     The margin parameter is optional. When given, it indicates the amount of pixels around the given area that should be made visible as well. */
-    scrollIntoView(
-      pos: { left: number; top: number; right: number; bottom: number },
-      margin: number
-    ): void;
-
-    /** Scrolls the given element into view. pos is a { line, ch } object, in editor-local coordinates.
-     The margin parameter is optional. When given, it indicates the amount of pixels around the given area that should be made visible as well. */
-    scrollIntoView(pos: Position, margin?: number): void;
-
     /** Scrolls the given element into view. pos is a { from, to } object, in editor-local coordinates.
      The margin parameter is optional. When given, it indicates the amount of pixels around the given area that should be made visible as well. */
     scrollIntoView(pos: Range, margin?: number): void;
```

Comment:
The typing for scrollIntoView should be deleted - I think the code does not take a Codemirror `Range` object, but rather the signature at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/89592f7ec313174363136fe91749cedff272a70e/types/codemirror/index.d.ts#L294 is what is needed.